src/Roslyn/Core/ApiDesign/DeclarePublicAPIFix.cs

```diff
@@ -129,6 +129,14 @@ protected override Task<Solution> GetChangedSolutionAsync(CancellationToken canc
             {
                 return _createChangedAdditionalDocument(cancellationToken);
             }
+
+            public override string EquivalenceKey
+            {
+                get
+                {
+                    return nameof(AdditionalDocumentChangeAction);
+                }
+            }
```

Comment:
Yes. This is addressing a Roslyn rule RS1011. "... has the default value of 'null' for property 'EquivalenceKey'. Either override this property on '...' to return a non-null and unique value across all code actions per-fixer or use such an existing code action.	

Will remove it for now since it is not related to the purpose of this pull request.